test/index.test.js

```diff
@@ -283,6 +283,18 @@ test('AWS.mock function should mock AWS service and method on the service', func
     st.end();
   });
   t.end();
+  t.test('Restore should not fail when the stub did not exist.', function (st) {
+    // This test will fail when restoring throws unneeded errors.
+    try {
+      var stub = awsMock.mock('CloudSearchDomain', 'search');
+      awsMock.restore('SES', 'sendEmail');
+      awsMock.restore('CloudSearchDomain', 'doesnotexist');
+      st.end();
```

Comment:
do we need an _assertion_ here? or is the fact that the error is _not_ thrown enough?